node/pkg/watchers/evm/chain_config.go

```diff
@@ -128,28 +132,28 @@ var (
 		// As of 11/10/2023 Scroll supports polling for finalized but not safe.
 		vaa.ChainIDScroll: {Finalized: true, Safe: false, EvmChainID: 534351, PublicRPC: "https://scroll-sepolia-rpc.publicnode.com"},
 
-		vaa.ChainIDMantle: {Finalized: true, Safe: true, EvmChainID: 5003, PublicRPC: "https://mantle-sepolia.drpc.org"},
-		vaa.ChainIDBlast:  {Finalized: true, Safe: true, EvmChainID: 168587773, PublicRPC: "https://blast-sepolia.drpc.org"},
+		vaa.ChainIDMantle: {Finalized: true, Safe: true, EvmChainID: 5003, PublicRPC: "https://rpc.sepolia.mantle.xyz"},
+		vaa.ChainIDBlast:  {Finalized: true, Safe: true, EvmChainID: 168587773, PublicRPC: "https://sepolia.blast.io"},
 		vaa.ChainIDXLayer: {Finalized: true, Safe: true, EvmChainID: 195, PublicRPC: "https://xlayertestrpc.okx.com"},
 
 		// As of 9/06/2024 Linea supports polling for finalized but not safe.
-		vaa.ChainIDLinea: {Finalized: true, Safe: false, EvmChainID: 59141, PublicRPC: "https://linea-sepolia-rpc.publicnode.com"},
+		vaa.ChainIDLinea: {Finalized: true, Safe: false, EvmChainID: 59141, PublicRPC: "https://rpc.sepolia.linea.build"},
 
-		vaa.ChainIDBerachain: {Finalized: false, Safe: false, EvmChainID: 80084, PublicRPC: "https://bartio.rpc.berachain.com"},
+		vaa.ChainIDBerachain: {InstantFinality: true, Finalized: true, Safe: true, EvmChainID: 80084, PublicRPC: "https://bartio.rpc.berachain.com"},
```

Comment:
```suggestion
		vaa.ChainIDBerachain: {InstantFinality: true, Finalized: true, Safe: true, EvmChainID: 80084, PublicRPC: "https://berachain-testnet-rpc.publicnode.com/"},
```

Bartio has been deprecated

Reply:
I don't think it has? I can still access it from the explorer and read the guardian set from the core contract.